$OpenBSD$

nuke pandoc dependency
smash bundled libclang & replace with ports clang dirs

Index: src/cpp/session/CMakeLists.txt
--- src/cpp/session/CMakeLists.txt.orig
+++ src/cpp/session/CMakeLists.txt
@@ -25,9 +25,6 @@ endif()
 if(NOT EXISTS "${RSTUDIO_DEPENDENCIES_DIR}/common/mathjax-26")
   message(FATAL_ERROR "Mathjax 2.6 not found (re-run install-dependencies script to install)")
 endif()
-if(NOT EXISTS "${RSTUDIO_DEPENDENCIES_DIR}/common/pandoc")
-  message(FATAL_ERROR "pandoc not found (re-run install-dependencies script to install)")
-endif()
 
 # we often install embedded versions of these packages but don't do so currently
 
@@ -38,22 +35,7 @@ endif()
 #   message(FATAL_ERROR "rsconnect package not found (re-run install-dependencies script to install)")
 # endif()
 
-# verify libclang is installed
-if(WIN32)
-   set(LIBCLANG_VERSION "3.4")
-else()
-   set(LIBCLANG_VERSION "3.5")
-endif()
-set(LIBCLANG_DIR "${RSTUDIO_DEPENDENCIES_DIR}/common/libclang/${LIBCLANG_VERSION}")
-if(NOT EXISTS "${LIBCLANG_DIR}")
-  message(FATAL_ERROR "libclang ${LIBCLANG_VERSION} not found  (re-run install-dependencies script to install)")
-endif()
-set(LIBCLANG_HEADERS_DIR "${RSTUDIO_DEPENDENCIES_DIR}/common/libclang/builtin-headers")
-if(NOT EXISTS "${LIBCLANG_HEADERS_DIR}")
-  message(FATAL_ERROR "libclang builtin-headers not found  (re-run install-dependencies script to install)")
-endif()
 
-
 # include files
 file(GLOB_RECURSE SESSION_HEADER_FILES "*.h*")
 
@@ -502,19 +484,8 @@ if (NOT RSTUDIO_SESSION_WIN64)
    install(DIRECTORY "resources/profiler"
            DESTINATION ${RSTUDIO_INSTALL_SUPPORTING}/resources)
 
-   # install libclang
-   if(WIN32)
-      file(GLOB LIBCLANG_32_FILES "${LIBCLANG_DIR}/x86/libclang.*")
-      install(PROGRAMS ${LIBCLANG_32_FILES}
-              DESTINATION  ${RSTUDIO_INSTALL_BIN}/rsclang/x86)
-      file(GLOB LIBCLANG_64_FILES "${LIBCLANG_DIR}/x86_64/libclang.*")
-      install(PROGRAMS ${LIBCLANG_64_FILES}
-              DESTINATION  ${RSTUDIO_INSTALL_BIN}/rsclang/x86_64)
-   else()
-      file(GLOB_RECURSE LIBCLANG_FILES "${LIBCLANG_DIR}/libclang.*")
-      install(PROGRAMS ${LIBCLANG_FILES}
-              DESTINATION  ${RSTUDIO_INSTALL_BIN}/rsclang)
-   endif()
+   set(LIBCLANG_DIR "/usr/local/lib")
+   set(LIBCLANG_HEADER_DIR "/usr/local/include/clang-c")
 
    # install libclang builtin-headers
    install(DIRECTORY "${RSTUDIO_DEPENDENCIES_DIR}/common/libclang/builtin-headers"
